pages/index.js

```diff
@@ -1,65 +1,67 @@
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import Link from 'next/link';
+import { USERS } from './api/constants/users';
+import { TAGS } from './api/constants/tags';
+import { BUILDS } from './api/constants/builds';
+
+// TODO(Renzo): Implement properly styled nav bar
+// TODO(Renzo): Implement data fetching for tags, posts, users, etc.
 
 export default function Home() {
   return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
+    <div>
+      <Link href="/">
+        <a>
+          <h1>Learn Build</h1>
+        </a>
+      </Link>
 
-        <div className={styles.grid}>
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h3>Documentation &rarr;</h3>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
+      <h2>Nav Bar</h2>
+      <ul>
+        <li>
+          <Link href="/about">
+            <a>About</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/search">
+            <a>Search</a>
+          </Link>
+        </li>
+      </ul>
 
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h3>Learn &rarr;</h3>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
+      <h2>Builds</h2>
+      <ul>
+        {BUILDS.map(build => (
+          <li>
+            <Link href="/build/[name]" as={`/build/${build.name}`}>
+              <a>{build.name}</a>
+            </Link>
+          </li>
+        ))}
+      </ul>
 
-          <a
-            href="https://github.com/vercel/next.js/tree/master/examples"
-            className={styles.card}
-          >
-            <h3>Examples &rarr;</h3>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
+      <h2>Users</h2>
+      <ul>
+        {USERS.map(user => (
+          <li>
+            <Link href="/[user]" as={`/${user.name}`}>
```

Comment:
Just wondering, why are users under `/:user` and tags under `/tags/:tag`

Reply:
Product Hunt, Instagram, and Twitter all have user profiles under `/:user`, so I figured for familiarity reasons I would follow the same format. By the same lines, Instagram/Twitter and Product Hunt use hashtags/topics under `/hashtag/:hashtag` and `/topic/:topic` respectively. Let me know if you think that's weird though :sweat_smile: